apps/omg/lib/omg/supervisor.ex

```diff
@@ -28,7 +28,15 @@ defmodule OMG.Supervisor do
     children = [
       {OMG.InternalEventBus, []},
       {OMG.EthereumClientMonitor, [alarm_module: Alarm]},
-      {OMG.EthereumHeight, []}
+      {OMG.EthereumHeight, []},
+      {SpandexDatadog.ApiServer,
```

Comment:
Spandex uses this `ApiServer` to send requests to Datadog asynchronously. We have many supervisors and I'm still digging into where this should live, but tests pass when I put it here so it seemed like a good start. 😆

Reply:
hmm... maybe omg_status isn't a bad place for this!